src/seedu/addressbook/commands/order/DraftOrderConfirmCommand.java

```diff
@@ -31,9 +31,11 @@ public CommandResult execute() {
             String message;
             if (draftOrder.hasDishItems()) {
                 final ReadOnlyMember customerOfOrderToAdd = draftOrder.getCustomer();
-                final Order toAdd = new Order(customerOfOrderToAdd, draftOrder.getDishItems());
+                final Order toAdd = new Order(customerOfOrderToAdd, draftOrder.getDishItems(), draftOrder.getPoints());
+                final int pointsToRedeem = draftOrder.getPoints();
+                final double finalPrice = toAdd.getPrice() - toAdd.getPoints();
```

Comment:
Rejected.
Calculation should be defined in method calculatePrice() in Order.java

Reply:
I can handle that one